src/components/Shared/Buttons.js

```diff
@@ -223,6 +223,16 @@ export const getButtons = platformLabels => [
     image: 'clear.png',
     main: true
   },
+  {
+    key: 'generate',
+    label: platformLabels.Generate,
+    condition: 'isGenerated',
+    onClick: 'onGenerate',
+    disabled: '!editMode',
+    color: 'black',
+    image: 'generate.png',
+    main: true
```

Comment:
it's not main button, main should be false

Reply:
is this fixed??